cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -155,6 +155,7 @@ static gpi_objtype_t const_type_to_gpi_objtype(int32_t const_type) {
     switch (const_type) {
         case vpiDecConst:
         case vpiBinaryConst:
+            return GPI_BITS;
         case vpiOctConst:
         case vpiHexConst:
```

Comment:
Surely you want to include these too?

Reply:
Yes I thought so too, but wasn't sure because I could not think of a Verilog construct that generates any of these. Do You know how to construct an Oct or Hex constant in verilog? I guess constant means it must a `parameter` or `localparam`, but how to make them 'octal' or 'hex'?

Reply:
`localparam HEX_PARAM = 16'hABCD;` does not become `vpiHexConst` and that would be my only idea how to generate one.

Reply:
I would guess that parameters (that are not of `integer` type) initialized with `8’hA` end up of type Hex, in some simulators?